lib/rubocop/cop/mixin/multiline_literal_brace_layout.rb

```diff
@@ -38,16 +38,20 @@ def autocorrect(node)
 
       private
 
+      def children(node)
+        node.children
```

Comment:
On a method call (`on_send`) the value of `node.children` is actually the receiver, method name and then arguments. For instance, parsing

```
something.foo(1,2,3)
```

returns (pseudo-code)

```
[something, :foo, 1, 2, 3]
```

This is not the case for all other types (hash, array, method definitions) which only return hash elements, array elements or method parameters. This method allows the same code to be re-used for method calls.